gazebo/physics/HeightmapShape.cc

```diff
@@ -116,7 +116,16 @@ int HeightmapShape::LoadTerrainFile(const std::string &_filename)
       ignition::math::Angle latitude, longitude;
       double elevation;
 
-      this->dem.GetGeoReferenceOrigin(latitude, longitude);
+      try
+      {
+        this->dem.GetGeoReferenceOrigin(latitude, longitude);
+      }
+      catch(const common::Exception &_e)
```

Comment:
`_e` isn't used, I think it could be omitted, or you could print its message.

Reply:
I'll remove `_e`; I had been intending to print the exception message, but then I realized that `gzthrow` already prints the message:

* [gzthrow creates an Exception with the multi-argument constructor](https://github.com/osrf/gazebo/blob/gazebo11/gazebo/common/Exception.hh#L39-L41)
* [that constructor calls `Exception::Print`](https://github.com/osrf/gazebo/blob/gazebo11/gazebo/common/Exception.cc#L36-L41)

Reply:
Oh nice :+1:

Reply:
removed `_e` in d217b05